app/src/main/java/com/puzzlebench/clean_marvel_kotlin/data/provider/ContentLoader.kt

```diff
@@ -0,0 +1,56 @@
+package com.puzzlebench.clean_marvel_kotlin.data.provider
+
+import android.app.LoaderManager
+import android.content.CursorLoader
+import android.content.Loader
+import android.database.Cursor
+import android.os.Bundle
+import com.puzzlebench.clean_marvel_kotlin.data.provider.CharacterProvider.Companion.CONTENT_URI
+import com.puzzlebench.clean_marvel_kotlin.data.provider.CharacterProvider.Companion.DESCRIPTION_COLUM
+import com.puzzlebench.clean_marvel_kotlin.data.provider.CharacterProvider.Companion.ID_COLUM
+import com.puzzlebench.clean_marvel_kotlin.data.provider.CharacterProvider.Companion.NAME_COLUM
+import com.puzzlebench.clean_marvel_kotlin.data.provider.CharacterProvider.Companion.THUMBNAIL_EXTENSION_COLUM
+import com.puzzlebench.clean_marvel_kotlin.data.provider.CharacterProvider.Companion.THUMBNAIL_PATH_COLUM
+import com.puzzlebench.clean_marvel_kotlin.domain.model.Character
+import com.puzzlebench.clean_marvel_kotlin.domain.model.Thumbnail
+import com.puzzlebench.clean_marvel_kotlin.presentation.MainActivity
+import com.puzzlebench.clean_marvel_kotlin.presentation.mvp.view.CharacterView
+
+class ContentLoader(val context: MainActivity, val view: CharacterView) : LoaderManager.LoaderCallbacks<Cursor> {
```

Comment:
this is a loader specifically for Characters, I would rename it to: CharactersLoader or similar